_posts/2016-04-09-linux-mint-razer-blade.markdown

```diff
@@ -0,0 +1,97 @@
+
+###Prologue
+I've been looking for a portable gaming PC for a while. Last year, [Lifehacker's review of the 2015 Razer Blade Full HD](http://gizmodo.com/razer-blade-2015-review-finally-living-the-thin-gaming-1689073179) convinced me that was the machine I've been waiting for.
+
+A few months back Razer finally put that model up <a  href="http://www.amazon.com/gp/product/B01AGINE7Q/ref=as_li_tl?ie=UTF8&camp=1789&creative=9325&creativeASIN=B01AGINE7Q&linkCode=as2&tag=likethemammal-20&linkId=VLVDTKUEZSESNTID">on Amazon</a><img src="http://ir-na.amazon-adsystem.com/e/ir?t=likethemammal-20&l=as2&o=1&a=B01AGINE7Q" width="1" height="1" border="0" alt="" style="border:none !important; margin:0px !important;" />, at a discount. With the power of a credit card, I finally had the mini gaming workstation of my dreams in hand!
+
+I've done 3 or 4 full builds of Linux before, all with Mint. One on a Chromebook, but mostly on machines with slightly dated hardware. This build has by far been the hardest, and I learned the hard way that Linux isn't as stable on "new" hardware.
+
+###Chapter 1: Pre-install
+**Expand SSD storage (optional)**
+
+The SSD in the Full HD edition is about 250 GB. If you're dual-booting, like I did, this doesn't leave much space to split. If you're going to replace the ssd with a larger one, now's the time to do it. Heres a video on how:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/xjM-ByOE0fQ" frameborder="0" allowfullscreen></iframe>
+
+**Prepare your SSD (optional)**
+
+Theres loads of stuff that can be done to reduce the wear on a SSD. Several things on [this list](https://sites.google.com/site/easylinuxtipsproject/ssd) can, and should, only be done before install. Its something to consider.
+
+**Get bootable USB drive**
+
+Get the download for Mint from the [downloads page](https://www.linuxmint.com/download.php), I **highly recommend** you choose the Mate version over Cinnamon. This will come into play later. Although, if you choose Cinnamon and run into problems, there're ways to switch after the fact.
+
+Open up the Linux Mint [User Guide](https://www.linuxmint.com/documentation.php) and follow the instructions to verify your MD5. You'll also want this open for details on most of the initial setup process. This post only covers steps beyond the guide.
+
+Use [this tutorial](https://community.linuxmint.com/tutorial/view/744) to mount the downloaded `.iso` to a USB drive thats at least 2GB.
+
+###Chapter 2: Install
+Plug the Razer Blade into a power source. Follow the directions in the user guide to boot from USB and start the install process.
+
+If you end up on the partition menu to customize the size of the install, make sure to add a few GB partition as "swap area". Among other things, this is important for resuming from a suspend. Learned this the hard way.
```

Comment:
I'm going to leave this at "a few" cus I don't wanna be specific. Some places say 10Gbs, some say you don't need more than 2 or 3. This post isn't really about swap space, I just know if you don't have any suspend will get fucked up.